_includes/configuration/relative-urls.md

```diff
@@ -1,6 +1,9 @@
 ## relative_urls
 
-If this option is set to true, all URLs returned from the MCFileManager will be relative from the specified [document_base_url](#document_base_url). If it's set to false all URLs will be converted to absolute URLs.
+For URLs with the same domain as the page containing the {{site.productname}} editor. If this option accepts the following values:
```

Comment:
The last sentence reads rather strangely. Im not sure if should be there?

Reply:
Should be "If set to:" I think?

Reply:
fixed